RealEstate/apps/core/views.py

```diff
@@ -338,7 +384,9 @@ def post(self, request, *args, **kwargs):
                 Realtor.objects.create(user=user)
             user = authenticate(email=email, password=password)
             _login(request, user)
+            messages.success(request, "Realtor welcome message")
```

Comment:
I know this is a placeholder, but let's put something workable here just in case we don't get back to it.